pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/TestKitUtil.java

```diff
@@ -82,36 +78,36 @@ public class TestKitUtil {
             "INSERT INTO  sub_kits_settings (ddp_kit_request_settings_id, kit_type_id, kit_count, hide_on_sample_pages, external_name) "
                     + " VALUES (?, ?, 1, ?, '') ;";
     private static final String SELECT_DSM_KIT_REQUEST_ID = "SELECT dsm_kit_request_id from ddp_kit_request where ddp_kit_request_id like ? ";
-    private static final UserAdminTestUtil cmiAdminUtil = new UserAdminTestUtil();
```

Comment:
Keeping so many things in static state while also having a constructor that sets some of the static state makes it hard to keep the state correct, which resulted in some cryptic errors in circleCI with `Realm and study group already initialized`, which came from `UserAdminTestUtil.createRealmAndStudyGroup()` because it was a singleton here.  Running a few tests locally never uncovered this issue, but because circleCI runs a pile of tests in a single VM, the issue was exposed only in circleCI.  The changes here move most things into instance state, so if one test fails to clean up after itself, it should only impact that one test.